functions/system.bash

```diff
@@ -471,6 +471,26 @@ memory_split() {
   else
     if echo "gpu_mem=16" >> /boot/config.txt; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
+
+  sed -i '/^dtoverlay=vc4-fkms-v3d/d' /boot/config.txt
+}
+
+## disable or enable framebuffer to provide the maximum amount of memory for Linux operations.
+##
+##    use_framebuffer()
+##
+use_framebuffer() {
+  if ! is_pi; then return 0; fi
```

Comment:
Probably should make a clean exit statement if interactive on a non RPi system so the user knows why they just got booted back to the menu instead of where they expected

Reply:
ok